components/brave_wallet/common/eth_request_helper_unittest.cc

```diff
@@ -316,8 +316,71 @@ TEST(EthResponseHelperUnitTest, ParsePersonalSignParams) {
   EXPECT_FALSE(ParsePersonalSignParams(json, nullptr, nullptr));
   EXPECT_FALSE(
       ParsePersonalSignParams("{\"params\":[{}]}", &address, &message));
```

Comment:
we should also add `ParsePersonalEcRecoverParams` to `EthRequestHelperUnitTest.CommonParseErrors`

Reply:
Fixed in rebase https://github.com/brave/brave-core/pull/11590/commits/83d9229f9bd1c95362238c415c7c02f7f6f6c868